server/src/schemas/storeFrontSchema.ts

```diff
@@ -5,64 +5,73 @@ export const StoreFrontTypeDefs = gql`
   scalar Date
 
   type InventoryItem {
-    id: ID!;
-    image: String;
-    name: String;
-    labels: [String];
-    unit: String;
-    pluralUnit: String;
-    count: Int;
-    pricePerUnit: Float;
+    id: ID!
+    image: String
+    name: String
+    labels: [String]
+    unit: String
+    pluralUnit: String
+    count: Int
+    pricePerUnit: Float
   }
 
   input InventoryItemInput {
-    image: String;
-    name: String;
-    labels: [String];
-    unit: String;
-    pluralUnit: String;
-    count: Int;
-    pricePerUnit: Float;
+    image: String
+    name: String
+    labels: [String]
+    unit: String
+    pluralUnit: String
+    count: Int
+    pricePerUnit: Float
   }
 
   type PurchaseOrderItem {
-    id: ID!;
-    item: InventoryItem;
-    count: Int;
+    id: ID!
+    item: InventoryItem
+    count: Int
   }
 
   type PurchaseOrder {
-    id: ID!;
-    creator: Person;
-    createDate: Date;
```

Comment:
These are all dates at the moment, should they be datetimes instead?

Reply:
I think in general we should default to `DateTime` in the backend. If the user doesn't need to know the time of an event, that's something we can handle in the frontend.